net/coap/client.go

```diff
@@ -470,6 +479,18 @@ func DialUDP(ctx context.Context, addr string, opts ...DialOptionFunc) (*ClientC
 	if cfg.KeepAlive != nil {
 		dopts = append(dopts, udp.WithKeepAlive(cfg.KeepAlive))
 	}
+	if cfg.errors != nil {
+		dopts = append(dopts, udp.WithErrors(cfg.errors))
+		dopts = append(dopts, udp.WithGoPool(func(f func() error) error {
+			go func() {
```

Comment:
Is the new coroutine necessary here? Should not this be a responsibility of the callback implementation to be swift or fire a new coroutine?

Reply:
`WithGoPool` is not needed to set for handle errors